app/assets/json/translations/react/en.json

```diff
@@ -1118,12 +1118,15 @@
   "listings.lotteryPreference.Certificate of Preference (COP).desc": "For households in which at least one member holds a Certificate of Preference from the former San Francisco Redevelopment Agency. COP holders were displaced by Agency action generally during the 1960s and 1970s.",
   "listings.lotteryPreference.Certificate of Preference (COP).readMore": "Read more about the Certificate of Preference",
   "listings.lotteryPreference.Certificate of Preference (COP).title": "Certificate of Preference (COP)",
+  "listings.lotteryPreference.Certificate of Preference.title": "Certificate of Preference (COP)",
```

Comment:
Yeah, I've gone back and forth on this. The two options: Refactor the stubbed data to add the acronym, or add the non-acronym string into the en.json. The reason I chose the latter was because the stubbed data seemed to have come from somewhere, and adding to the en.json seemed like a low cost way to protect against the possibility that in the application we are getting a non acronym preference string. What do you think?

Reply:
Sounds good! Will do that now!

Reply:
There is no direct test to check for non-`.title`  strings. I added those just to be safe (and since we have the translations already), so some probably aren't being used or directly tested